Server/Content/Sources/App/Models/Text.swift

```diff
@@ -0,0 +1,27 @@
+import Fluent
+import Vapor
+
+final class Text: Model, Content {
+    static let schema = "texts"
+    
+    @ID(key: .id)
+    var id: UUID?
+
+    @Field(key: "title")
+    var title: String
+    
+    @Field(key: "text")
+    var text: String
+    
+    @Field(key: "backward_text")
+    var backwardText: String?
+
+    init() { }
+
+    init(id: UUID? = nil, title: String, text: String, backwardText: String? = nil) {
```

Comment:
Are default parameters are necessary? 
```swift 
init(id: UUID? = nil...
```
Can't we make it just:
```swift
init(id: UUID?, ...
```

Reply:
No, I think we can't. Because, on `create` POST request of model controller decodes request body and creates model instance right away, thus means model should be able to be created without `id` property, because `id` will be created and assigned by database